### PR TITLE
530 claimant contact confirmation email

### PR DESCRIPTION
**Pull request by cehsu**
Connects to department-of-veterans-affairs/vets.gov-team#3152
screen shot 2017-06-15 at 5 09 58 pm

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -8300,9 +8300,9 @@ verror@1.3.6:
   dependencies:
     extsprintf "1.0.2"
 
-"vets-json-schema@https://github.com/department-of-veterans-affairs/vets-json-schema.git#175be45a2005e1d2ade7c43419d3dd0ca82d6c07":
-  version "3.0.3"
-  resolved "https://github.com/department-of-veterans-affairs/vets-json-schema.git#175be45a2005e1d2ade7c43419d3dd0ca82d6c07"
+"vets-json-schema@https://github.com/department-of-veterans-affairs/vets-json-schema.git#b4c9f42f7f1182952d59bb23db5867ed1eb2c848":
```
Comment (cehsu) [inline yarn.lock]: This is not the merged commit hash.

**Review by cehsu — COMMENTED** [review]
Must update commit hash for vets-json-schema dependency.

File: src/js/burials/config/form.js [hunk 2]
```diff
@@ -153,18 +155,29 @@ const formConfig = {
           title: 'Claimant Contact Information',
           path: 'claimant-contact-information',
           uiSchema: {
+            'ui:validation': validateMatch('claimantEmail', 'view:claimantConfirmationEmail'),
             claimantAddress: address.uiSchema('Address'),
             claimantEmail: {
-              'ui:title': 'Email address'
+              'ui:title': 'Email address',
+              'ui:errorMessages': {
+                pattern: 'Please put your email in this format x@x.xxx'
```
Comment (jbalboni) [inline src/js/burials/config/form.js]: Does there need to be a colon after format?

File: src/js/burials/config/form.js [hunk 3]
```diff
@@ -153,18 +155,29 @@ const formConfig = {
           title: 'Claimant Contact Information',
           path: 'claimant-contact-information',
           uiSchema: {
+            'ui:validation': validateMatch('claimantEmail', 'view:claimantConfirmationEmail'),
```
Comment (jbalboni) [inline src/js/burials/config/form.js]: I think we could add a test to make sure there's a validation error when these don't match. To keep us from forgetting to change the property names here later if we make a change.

**Review by jbalboni — APPROVED** [review]
Looks good, had one minor question and I think it needs one more test.

**Comment by cehsu** [comment]
Great idea @jbalboni, test added.

**Pull request merged**